frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -517,19 +518,32 @@ export const NamespacedClusterServiceVersionList: React.SFC<ClusterServiceVersio
   );
   const NoDataEmptyMsg = () => <MsgBox title="No Operators Found" detail={noDataDetail} />;
 
-  const isCopiedCSV = (source, kind) => {
+  const isCopiedCSV = (source: ClusterServiceVersionKind | SubscriptionKind, kind: string) => {
     return (
       referenceForModel(ClusterServiceVersionModel) === kind && source.status?.reason === 'Copied'
     );
   };
 
-  const removeCopiedCSVs = (operators: (ClusterServiceVersionKind | SubscriptionKind)[]) => {
+  const isStandardCSV = (operator: ClusterServiceVersionKind | SubscriptionKind) => {
+    return (
+      operator.metadata.annotations?.[operatorTypeAnnotation] !== nonStandardAnnotationValue &&
+      operator.status?.phase !== ClusterServiceVersionPhase.CSVPhaseFailed
```

Comment:
Won't this hide any failed operator from the list? We want to hide CSVs with the annotation unless they're failed.

```suggestion
      operator.metadata.annotations?.[operatorTypeAnnotation] !== nonStandardAnnotationValue ||
      operator.status?.phase === ClusterServiceVersionPhase.CSVPhaseFailed
```

Reply:
So I was in the impression that we want to hide only